File: Simplesort/simplesort.js
function simple_sort(values){

    for(let i = 0; i < values.length; i++){
        for(let j = i+1; j < values.length; j++){
            if(values[j] < values[i]){
                let temp = values[i];
                values[i] = values[j];
                values[j] = temp
            }
        }
    }

    return values;
}

File: Simplesort/simplesort.py
def simple_sort(values):

    for i in range(0, len(values)):
        for j in range(i + 1, len(values)):
            if values[j] < values[i]:
                temp = values[j]
                values[j] = values[i]
                values[i] = temp

    return values